### DISCUSSION TITLE
npx prisma generate gives the download error

### DISCUSSION THREAD

**Discussion by shivpratikhande** [prisma/prisma#24125] [Q&A] [answered]
### Bug description

![Screenshot from 2024-05-08 18-45-20](https://github.com/prisma/prisma/assets/127005505/c06578a7-b82a-4181-a276-b29824a77435)

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by janpio** [comment]
Can you copy past that link into the browser to download the file?
If so, nothing is wrong on our side but somehow your system blocking the request. See here for more information on the error it is returning: https://stackoverflow.com/questions/40182121/whats-the-cause-of-the-error-getaddrinfo-eai-again

**Reply by shivpratikhande** [reply, marked as answer]
There was DNS error from my pc, it got resolved...
